### DISCUSSION TITLE
Should redemptions using chat commands be implemented?

### DISCUSSION THREAD

**Discussion by skarab42** [skarab42/ValheimTwitch#7] [Q&A]
This mod relies on channel points which are a feature only available to Twitch affiliates & partners. 
If you're interested in a version using chat commands, show me your interest by upvoting the following issue.

**Comment by Yaume7** [comment]
Ce serait vraiment super d'avoir cette fonction !

**Comment by BRN-5** [comment]
nice mod

**Comment by wizen** [comment]
If you don't do it I will :)

**Comment by Cortidd** [comment]
please yes, I'd love to use it but I'm only just starting on twitch!

**Comment by devildos** [comment]
This would be amazing to see. i was trying to get this mod running before but failed attempting getting things done.

**Comment by NorgerLegacy** [comment]
With so many small twitch streamers and valheim losing player base this would help boost the mod but also the small streamers trying to get out there and have something different then just the game alone and have the viewers interact. 

i'd suggest using the BTI bethesda twitch integration method of a twitch bot to save a file it reads on the streamers pc that saves each viewer and points earned when the bot is run, and a script mod to facilitate the various commands setup for users in the chat.
